README.md

```diff
@@ -23,6 +23,7 @@ SDK covers all of Klarna API: https://developers.klarna.com/api/
 ### Prerequisites
 * PHP 5.5 or above
 * [API credentials](#api-credentials)
+* [Composer](https://getcomposer.org/) (optional)
```

Comment:
Proposal -> "(optional if you want to use 3rd-party HTTP transport Libraries, like for example Guzzle).."

Reply:
I don't like the information overloading in this place. We have all the required info below.